$OpenBSD: patch-uip_mhbuildsbr_c,v 1.1 2013/11/14 07:36:27 bentley Exp $

Fix for mhshow-charset- directives in .mh_profile being broken
http://git.savannah.gnu.org/cgit/nmh.git/commit/?id=b5f46f4ba364af010a7ef5afb20b667fcbe9b136


--- uip/mhbuildsbr.c.orig	Sun Jun 10 22:06:19 2012
+++ uip/mhbuildsbr.c	Mon Oct 28 01:09:15 2013
@@ -1314,12 +1314,11 @@ scan_content (CT ct)
 		continue;
 
 	    if (contains8bit) {
-		t->tx_charset = CHARSET_UNKNOWN;
 		*ap = concat ("charset=", write_charset_8bit(), NULL);
 	    } else {
-		t->tx_charset = CHARSET_USASCII;
 		*ap = add ("charset=us-ascii", NULL);
 	    }
+	    t->tx_charset = CHARSET_SPECIFIED;
 
 	    cp = strchr(*ap++, '=');
 	    *ap = NULL;
$OpenBSD: patch-uip_mhbuildsbr_c,v 1.1 2013/11/14 07:36:27 bentley Exp $

Fix for mhshow-charset- directives in .mh_profile being broken
http://git.savannah.gnu.org/cgit/nmh.git/commit/?id=b5f46f4ba364af010a7ef5afb20b667fcbe9b136


--- uip/mhbuildsbr.c.orig	Sun Jun 10 22:06:19 2012
+++ uip/mhbuildsbr.c	Mon Oct 28 01:09:15 2013
@@ -1314,12 +1314,11 @@ scan_content (CT ct)
 		continue;
 
 	    if (contains8bit) {
-		t->tx_charset = CHARSET_UNKNOWN;
 		*ap = concat ("charset=", write_charset_8bit(), NULL);
 	    } else {
-		t->tx_charset = CHARSET_USASCII;
 		*ap = add ("charset=us-ascii", NULL);
 	    }
+	    t->tx_charset = CHARSET_SPECIFIED;
 
 	    cp = strchr(*ap++, '=');
 	    *ap = NULL;
